lib/src/main/java/graphql/nadel/engine/blueprint/NadelFieldInstruction.kt

```diff
@@ -76,6 +77,11 @@ interface NadelGenericHydrationInstruction {
      * The field definition in the overall schema referenced by [queryPathToActorField].
      */
     val actorFieldDef: GraphQLFieldDefinition
+
+    /**
+     * The container of the actor field in the overall schema referenced by [queryPathToActorField].
+     */
+    val actorFieldContainer: GraphQLFieldsContainer
```

Comment:
we put this in the blue print instruction - and then use it later during the hydration calls